<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kabinet</class>
 <widget class="QDialog" name="Kabinet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>586</width>
     <height>387</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Общие настройки</string>
    </attribute>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>571</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Использовать смайлы</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>32</y>
       <width>571</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Включить отладочные записи</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>54</y>
       <width>571</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Отображение ошибок</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>571</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Использовать виртуальный сервер для входа</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>102</y>
       <width>581</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Фильтровать нецензурные слова</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>127</y>
       <width>581</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Автоматический вход</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Сервера</string>
    </attribute>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Отображать в списке</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;b&gt;&lt;center&gt;Настройки</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;b&gt;&lt;center&gt;Информация</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>131</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_nameserver">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_url">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_region">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_serverstatus">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>431</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_serveronline">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_serverinfo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>421</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_serverpixmap">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>70</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Мой аккаунт</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;b&gt;Имя</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMyName">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;b&gt;Права</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Prem">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;font color=green&gt;Польователь</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;b&gt;Цвет</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMyColor">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;font color=#00F&gt;#00F</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>330</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cохранить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;b&gt;Часовой пояс:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>157</y>
       <width>162</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;b&gt;Префикс</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPrefix">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>57</y>
       <width>159</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_9">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>86</y>
       <width>221</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Статус Онлайн виден</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>Безопастность</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>111</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_ClientVersion_Uni">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_LastData_Uni">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_IP_Uni">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_Data_Uni">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label__UniKey_Uni">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>451</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>571</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;b&gt;&lt;center&gt;Список авторизированных компьютеров</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>330</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить списки</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>571</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Выключить повторную авторизацию(Безопастно)</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить ключ</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Мои чаты</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;b&gt;&lt;center&gt;Информация</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_NameKom">
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>20</y>
       <width>451</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_NumKom">
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>45</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_Creater">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>67</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CreateRoom">
     <property name="geometry">
      <rect>
       <x>141</x>
       <y>332</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать комнату</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>332</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Пригласить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>332</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить комнату</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Свободная комната: &lt;font color=red&gt;Нет</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>121</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>332</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Перейти</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Участники:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CreateRoom">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>290</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_CreateRoom">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>263</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;center&gt;Имя комнаты</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>151</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Участники</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>131</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;b&gt;&lt;center&gt;Информация</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_StatusUser">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>12</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_NameUser">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_PermissionUser">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_EnterClientUser">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_RegionUser">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_TomeZomeUser">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="laabel_TextStatus">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_TimeOnlineUser">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>Смайлы</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>581</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>581</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;center&gt;&lt;b&gt;Текущие смайлы</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>361</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>7</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_10">
    <attribute name="title">
     <string>Прокси</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>URL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Порт</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>16</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>45</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>77</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Логи</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>581</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>О программе</string>
    </attribute>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>581</width>
       <height>361</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;center&gt;Версия: 1.0.2&lt;br&gt;Версия ACore: 1.0.1&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;hr&gt;&lt;font color=red size=5&gt;Мультиплатформенный клиент Cluster-Chat&lt;/font&gt;&lt;br&gt;Разработчик клиента Gravit&lt;br&gt;Сервера предоставлены Ulin&lt;/font&gt;&lt;hr&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
